retrieverdash/dashboard_script/dashboard_script.py

```diff
@@ -0,0 +1,58 @@
+import json
+import os
+from multiprocessing import Pool
+
+from filelock import FileLock
+from retriever import datasets
+
+from status_dashboard_tools import (
+    get_dataset_md5,
+    diff_generator,
+    create_dirs,
+    dataset_to_csv)
+
+file_location = os.path.dirname(os.path.realpath(__file__))
+
+example_datasets = ['abalone-age', 'airports', 'biodiversity-response', 'bird-size',
```

Comment:
This is too long, P8P recommends nothing longer than 80, unless you compromise readability.

```python
example_datasets = [
    'abalone-age',
    'airports',
    'biodiversity-response',
    'bird-size',
    'breast-cancer-wi',
    'butterfly-population-network',
    'partners-in-flight',
    'portal']
``` 

or 

```
example_datasets = [
    'abalone-age', 'airports',
    'biodiversity-response', 'bird-size',
    'breast-cancer-wi', 'butterfly-population-network',
    'partners-in-flight', 'portal']
```